csvqb/csvqb/inputs.py

```diff
@@ -28,20 +28,39 @@ def pandas_input_to_columnar(
 
 
 def pandas_input_to_columnar_list(
-    maybe_columnar_data: PandasDataTypes, allow_none: bool = True
+    maybe_columnar_data: PandasDataTypes, allow_no_data_at_all: bool = True
 ) -> List[Any]:
-    series = pandas_input_to_columnar(maybe_columnar_data, allow_none)
+    series = pandas_input_to_columnar(maybe_columnar_data, allow_no_data_at_all)
     if series is None:
         return []
 
     return series.tolist()
 
 
-def pandas_input_to_columnar_str(
-    maybe_columnar_data: PandasDataTypes, allow_none: bool = True
-) -> Iterable[str]:
-    series = pandas_input_to_columnar(maybe_columnar_data, allow_none)
+def pandas_input_to_columnar_optional_str(
+    maybe_columnar_data: PandasDataTypes,
+    allow_no_data_at_all: bool = True,
+) -> Iterable[Optional[str]]:
+    series = pandas_input_to_columnar(maybe_columnar_data, allow_no_data_at_all)
```

Comment:
I know where this is being used but what is the actual usecase? We don't allow sparse columns.

Reply:
Just confirming, we do allow sparse attributes.